services/storage/src/simcore_service_storage/datcore_dsm.py

```diff
@@ -58,7 +58,32 @@ async def list_files(
         )
 
     async def get_file(self, user_id: UserID, file_id: StorageFileID) -> FileMetaData:
-        raise NotImplementedError
+        api_token, api_secret = await self._get_datcore_tokens(user_id)
+
+        package_files = await datcore_adapter.get_package_files(
+            self.app, api_token, api_secret, file_id
+        )
+        assert (
+            len(package_files) == 0
+        ), "More than one file in package, this breaks the current assumption"
+        resp_data = package_files[0]["content"]
+
+        return FileMetaData.construct(
```

Comment:
any particular reason to use `construct` here? i.e. why do you skip validation?

Reply:
@pcrespov: This is something I used already and here is the rationale why I did it so:
If I know exactly what I am using I do not need parsing/validation right? Therefore the use of .construct()

Reply:
I removed the call to `construct` (it was making  mypy tests on storage fail 😁 )